### PR TITLE
docs: categorize specs in readme

### PR DESCRIPTION
**Pull request by emschwartz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,43 +9,74 @@ The process for submitting an RFC is documented in the [RFC Process](CONTRIBUTIN
 
 For the main reference implementation of the ILP stack, see [Interledger.js](https://github.com/interledgerjs).
 
-## Index
-
-Here is a list of the main Interledger RFCs:
+## Interledger Overview and Explanatory Docs
```
Comment (sharafian) [inline README.md]: Maybe a short note here describing the ordering of these docs

File: README.md [hunk 2]
```diff
@@ -9,43 +9,74 @@ The process for submitting an RFC is documented in the [RFC Process](CONTRIBUTIN
...
+* **[3: Interledger Protocol (ILP)](0003-interledger-protocol/0003-interledger-protocol.md)**
 
-  The protocol used by connectors to communicate routing information.
+  Specifies the Interledger Protocol and Interledger Packet, which are used for sending payment instructions across different ledgers and connectors. This is the core protocol in the Interledger stack.
 
-* **[11: Interledger Payment Request (IPR)](0011-interledger-payment-request/0011-interledger-payment-request.md)**
+* **[8: Interledger Quoting Protocol (ILQP)](0008-interledger-quoting-protocol/0008-interledger-quoting-protocol.md)**
 
-  A Transport Layer protocol in which the receiver generates the ILP Packet and condition.
+  Specifies the protocol used for requesting a quote for an Interledger payment from a connector.
```
Comment (sharafian) [inline README.md]: worded a bit awkwardly

File: README.md [hunk 3]
```diff
@@ -9,43 +9,74 @@ The process for submitting an RFC is documented in the [RFC Process](CONTRIBUTIN
...
 
+## Protocols Built Upon ILP
+
+* **[9: Simple Payment Setup Protocol (SPSP)](0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md)**
+
+  A basic Application Layer protocol that uses HTTPS to exchange details needed to set up an Interledger payment.
+
+* **[11: Interledger Payment Request (IPR)](0011-interledger-payment-request/0011-interledger-payment-request.md)**
+
+  A Transport Layer protocol in which the receiver generates the ILP Packet and condition.
```
Comment (sharafian) [inline README.md]: protocol and data format

File: README.md [hunk 4]
```diff
@@ -9,43 +9,74 @@ The process for submitting an RFC is documented in the [RFC Process](CONTRIBUTIN
...
+
+* **[21: Plugin RPC API](0021-plugin-rpc-api/0021-plugin-rpc-api.md)**
+
+  Recommended API for trustlines and payment channels.
+
+## Routing
+
+* **[10: Connector-To-Connector Protocol (CCP)](0010-connector-to-connector-protocol/0010-connector-to-connector-protocol.md)**
+
+  The protocol used by connectors to communicate routing information.
```
Comment (sharafian) [inline README.md]: maybe put 'A protocol' instead of 'The protocol', because it's not being used everywhere

File: README.md (hunk 1 above)
Comment (emschwartz) [inline README.md]: Right now I just put them in numerical order within the groups, but that's not necessarily the best ordering

File: README.md (hunk 2 above)
Comment (emschwartz) [inline README.md]: Not sure how to change it. What would you suggest?

Comment (sharafian) [inline README.md]: maybe "The protocol for quoting an interledger payment, by sending a request to the connector"

File: README.md (hunk 3 above)
Comment (emschwartz) [inline README.md]: protocol implies data format too

**Pull request merged**